packages/ui/.storybook/main.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
     "@storybook/addon-links",
     "@storybook/addon-essentials",
     "@storybook/preset-create-react-app",
+    "@storybook/addon-controls",
```

Comment:
@AdrianValdes :) This was never deleted in the end

Reply:
But I do deleted `"@storybook/addon-controls"`, and I am checking my git and it is all synchronized

Reply:
I think I am understanding something wrong, sorry :(

Reply:
I thought we agreed on not deleting 
` "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@storybook/preset-create-react-app",`

Reply:
Yep maybe its me. We dont want to delete the default but only the
```
"@storybook/addon-controls",
```suggestion
```
```